cmd/http/server.go

```diff
@@ -172,13 +172,18 @@ var defaultCipherSuites = []uint16{
 	tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
 }
 
+// Go only provides constant-time implementations of Curve25519 and NIST P-256 curve.
+var secureCurves = []tls.CurveID{tls.X25519, tls.CurveP256}
+
 // NewServer - creates new HTTP server using given arguments.
 func NewServer(addrs []string, handler http.Handler, certificate *tls.Certificate) *Server {
 	var tlsConfig *tls.Config
 	if certificate != nil {
 		tlsConfig = &tls.Config{
 			PreferServerCipherSuites: true,
+			SessionTicketsDisabled:   true,
```

Comment:
Are you disabling session tickets in addition to some of the elliptic curves or is it a consequence of disabling some curves?

Reply:
Oh, sry forgot to remove after some experimentations. AFAIK we can disable them since they break PFS but we may want to keep that separated.